src/trans/gpu/algor/hicblas_gemm.hip.cpp

```diff
@@ -171,6 +170,12 @@ void run_group(Gemm &&gemm, int resol_id, int m, const int *n, const int *k,
     gemm(stream, m, n[i], k[i], alpha, A + offsetsA[i], lda, B + offsetsB[i],
          ldb[i], beta, C + offsetsC[i], ldc);
   }
+#ifdef CUDAGPU
+  HIC_CHECK(cudaStreamSynchronize(stream));
+#endif
+#ifdef HIPGPU
+  HIC_CHECK(hipStreamSynchronize(stream));
+#endif
```

Comment:
This `#ifdef` shouldn't be necessary if you add
```c
#define hipStreamSynchronize cudaStreamSynchronize
```
to [this file](https://github.com/PaulMullowney/ectrans/blob/amdflang-rework/src/trans/gpu/algor/hicblas_cuda.h). Same for the code above and in hicfft.hip.cpp.